File: iniciante/1002/1002.cpp
// Inclui a biblioteca padrão de entrada e saída
#include <iostream>

// Inclui a biblioteca para manipular a formatação da saída (como casas decimais)
#include <iomanip>

int main() // Função principal do programa
{
  // Define uma constante PI com valor aproximado de π
  const double PI = 3.14159;

  // Declara duas variáveis do tipo double: 'raio' e 'area'
  double raio, area;

  // Lê o valor do raio fornecido pelo usuário
  std::cin >> raio;

  // Calcula a área do círculo usando a fórmula: área = π * raio²
  area = PI * raio * raio;

  // Imprime a área no formato "A=valor" com 4 casas decimais
  std::cout << "A=" << std::fixed << std::setprecision(4) << area << std::endl;

  return 0; // Retorna 0 indicando que o programa terminou com sucesso
}

File: iniciante/1002/1002.py
# Define a constante PI com valor aproximado de π
PI = 3.14159

# Lê o valor do raio do usuário e converte para float
raio = float(input())

# Calcula a área do círculo usando a fórmula: área = π * raio²
area = PI * raio ** 2

# Imprime a área formatada com 4 casas decimais, no formato "A=xxxx.xxxx"
print(f"A={area:.4f}")
